src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,39 @@
+package step2.domain;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class Lotto {
+
+    public static final int NUMBER_SIZE = 6;
+
+    private final LottoNumber lottoNumbers;
+
+    public Lotto(LottoNumberGenerator lottoNumberGenerator) {
+        this.lottoNumbers = new LottoNumber(lottoNumberGenerator.getGeneratedNumbers());
+    }
```

Comment:
```suggestion
    public Lotto(Set<Integer> lottoNumbers) {
        this.lottoNumbers = lottoNumbers;
    }
```
이렇게만 표현해도 좋을 것 같습니다.
굳이 지금은 Generator를 주입할 필요는 없을 것 같아요!
아니면 Generator가 Lotto class를 만들어내는 방법도 있습니다.